Community/gsoc-alumni.md

```diff
@@ -184,4 +344,36 @@ The Manuscripts project, which is a part of the Grimoire Toolset, helps us in an
 
 #### Social Media
 
-* [LinkedIn](https://www.linkedin.com/in/pranjalaswani/)
\ No newline at end of file
+* [LinkedIn](https://www.linkedin.com/in/pranjalaswani/)
+
+
+## CHAOSS Outreachy students from May 2020 to August 2020 Interns
+
+|Student|Project|Mentors|
+|-------|:-----:|-------|
+|[Ore-Aruwaji Oloruntola](#Ore-Aruwaji-Oloruntola)|Build Workflow Process for CHAOSS Diversity & Inclusion Badging|Matt Germonprez, Matt Snell, Saleh Abdel Motaal|
```

Comment:
Please move the list of Outreachy students up to the same place that the GSoC lists are. 

I would prefer to have all the lists/tables together in one place in chronological order.